docs/orm-integration.md

```diff
@@ -193,21 +209,17 @@ Then modify your strategy's \_\_init__ function to match the following code:
             self._data_for_sale = self.collect_from_data_source()
         else:
             self._data_for_sale = kwargs.pop("data_for_sale", DEFAULT_DATA_FOR_SALE)
-
-        super().__init__(**kwargs)
-        self._oef_msg_id = 0
-
-        self._scheme = kwargs.pop("search_data")
-        self._datamodel = kwargs.pop("search_schema")
 ``` 
 
 At the end of the file modify the `collect_from_data_source` function : 
 ``` python
     def collect_from_data_source(self) -> Dict[str, Any]:
+        """Implement the logic to collect data."""
         connection = self._db_engine.connect()
         query = db.select([self._tbl])
         result_proxy = connection.execute(query)
-        return {"data": result_proxy.fetchall()}
```

Comment:
`result_proxy.fetchall()` returns a list of `RowProxy`. This caused a problem in the serialization of the Fipa.Inform message.

Moreover, we can't even use `list` in `info` messages, because Fipa requires a mapping from string to string: https://github.com/fetchai/agents-aea/blob/master/packages/fetchai/protocols/fipa/fipa.proto#L41